python/lsst/meas/extensions/scarlet/scarletDeblendTask.py

```diff
@@ -52,7 +50,7 @@
 
 __all__ = ["deblend", "ScarletDeblendConfig", "ScarletDeblendTask"]
 
-logger = lsst.log.Log.getLogger("meas.deblender.deblend")
+logger = logging.getLogger("lsst.meas.deblender.deblend")
```

Comment:
Wow, looks like this has had the wrong label ever since we ported this module from `lsst.meas.deblender.deblend.py`. The correct logging name should be `lsst.meas.extensions.scarlet.scarletDeblendTask`, so you should just be able to change this to `logging.getLogger("__name__")` like you did for the `source` module.